src/tests/readytorun/HardwareIntrinsics/X86/CpuId_R2R_Avx.csproj

```diff
@@ -17,6 +17,6 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <Compile Include="CpuId.cs" />
+    <Compile Include="../../../JIT/HardwareIntrinsics/X86/X86Base/CpuId.cs" />
```

Comment:
The local copy of these tests had been bit rotting for a while. I could bring it up to date, but it seems reasonable to just share it with JIT.